diff --git a/node_modules/@nejcm/docz-theme-extended/src/gatsby-theme-docz/index.js b/node_modules/@nejcm/docz-theme-extended/src/gatsby-theme-docz/index.js
index 19e4307..e82c5e7 100644
--- a/node_modules/@nejcm/docz-theme-extended/src/gatsby-theme-docz/index.js
+++ b/node_modules/@nejcm/docz-theme-extended/src/gatsby-theme-docz/index.js
@@ -1,22 +1,22 @@
 /* eslint-disable react/jsx-pascal-case */
-import {ComponentsProvider, theme, useConfig} from 'docz';
+import { ComponentsProvider, theme, useConfig } from 'docz';
 import baseComponents from 'gatsby-theme-docz/src/components';
 import defaultConfig from 'gatsby-theme-docz/src/theme';
-import {merge} from 'lodash/fp';
+import { merge } from 'lodash/fp';
 import React from 'react';
-import {Styled, ThemeProvider} from 'theme-ui';
+import { Themed, ThemeProvider } from 'theme-ui';
 
 const componentsMap = {
   ...baseComponents,
 };
 
 // eslint-disable-next-line react/prop-types
-const Theme = ({children}) => {
+const Theme = ({ children }) => {
   const config = useConfig();
   return (
     <ThemeProvider theme={config.themeConfig}>
       <ComponentsProvider components={componentsMap}>
-        <Styled.root>{children}</Styled.root>
+        <Themed.root>{children}</Themed.root>
       </ComponentsProvider>
     </ThemeProvider>
   );
