diff --git a/node_modules/gatsby-theme-docz/src/components/Layout/index.js b/node_modules/gatsby-theme-docz/src/components/Layout/index.js
index 89af5d4..d1ecf94 100644
--- a/node_modules/gatsby-theme-docz/src/components/Layout/index.js
+++ b/node_modules/gatsby-theme-docz/src/components/Layout/index.js
@@ -1,7 +1,7 @@
 /** @jsx jsx */
 import { useRef, useState } from 'react'
 import { jsx, Layout as BaseLayout, Main } from 'theme-ui'
-import { Global } from '@emotion/core'
+import { Global } from '@emotion/react'
 
 import global from '~theme/global'
 import { Header } from '../Header'
diff --git a/node_modules/gatsby-theme-docz/src/components/Sidebar/index.js b/node_modules/gatsby-theme-docz/src/components/Sidebar/index.js
index 9dafd5c..7725d45 100644
--- a/node_modules/gatsby-theme-docz/src/components/Sidebar/index.js
+++ b/node_modules/gatsby-theme-docz/src/components/Sidebar/index.js
@@ -1,7 +1,7 @@
 /** @jsx jsx */
 /** @jsxFrag React.Fragment */
 import React, { useState, useRef, useEffect } from 'react'
-import { Global } from '@emotion/core'
+import { Global } from '@emotion/react'
 import { jsx, Box } from 'theme-ui'
 import { useMenus, useCurrentDoc } from 'docz'
 
